word_racer/wordracer.py

```diff
@@ -0,0 +1,212 @@
+from redbot.core import commands
+import discord
+from .session import WordRacerSession
+from redbot.core import Config
+from redbot.core.i18n import Translator, cog_i18n
+from redbot.core.utils.chat_formatting import box, pagify
+
+UNIQUE_ID = 0xED5931AC
+
+_ = Translator("WordRacer", __file__)
+
+class WordRacer(commands.Cog):
+    def __init__(self):
+        super().__init__()
+        self.wordracer_sessions = []
+        self.conf = Config.get_conf(self, identifier=UNIQUE_ID, force_registration=True)
+        self.conf.register_member(wins=0, games=0, total_score=0)
+ 
+    @commands.group(invoke_without_command=True)
+    async def wordracer(self, ctx: commands.Context):
```

Comment:
Please include a docstring with a one line summary and at least a paragraph describing the game rules. (It will be automatically surfaced by `./help wordracer`)